sdk/src/types/api/plugins/indexer.rs

```diff
@@ -17,8 +17,11 @@ use crate::types::block::output::OutputId;
     serde(rename_all = "camelCase")
 )]
 pub struct OutputIdsResponse {
-    /// The ledger index at which the outputs were collected
-    pub ledger_index: u32,
+    /// The committed slot at which these outputs where available at.
```

Comment:
```suggestion
    /// The committed slot at which these outputs were available at.
```

Reply:
Please also comment on the TIP :trollface:

Reply:
Done